nncf/torch/hardware/fused_patterns.py

```diff
@@ -13,49 +27,148 @@
 from nncf.torch.graph.patterns import create_l2_norm
 
 
-def _get_torch_hw_fused_patterns() -> HWFusedPatterns:
-    retval = HWFusedPatterns()
-    linear_ops = GraphPattern()
-    linear_ops.add_node(**LINEAR_OPERATIONS)
-    retval.register(linear_ops, LINEAR_OPERATIONS['label'], match=False)
+PT_HW_FUSED_PATTERNS = Registry('torch')
 
-    batch_norm = GraphPattern()
-    batch_norm.add_node(**BATCH_NORMALIZATION_OPERATIONS)
-    retval.register(batch_norm, BATCH_NORMALIZATION_OPERATIONS['label'], match=False)
+# ATOMIC OPERATIONS
 
-    atomic_activations = GraphPattern()
-    atomic_activations.add_node(**ATOMIC_ACTIVATIONS_OPERATIONS)
-    swish = create_swish_act()
-    h_sigmoid = create_h_sigmoid_act()
-    h_swish = create_h_swish_act()
-    activations = atomic_activations | swish | h_swish | h_sigmoid
-    retval.register(activations, 'ACTIVATIONS', match=False)
 
-    arithmetic_ops = GraphPattern()
-    arithmetic_ops.add_node(**ARITHMETIC_OPERATIONS)
-    retval.register(arithmetic_ops, ARITHMETIC_OPERATIONS['label'], match=False)
+@PT_HW_FUSED_PATTERNS.register(PatternNames.L2_NORM)
+def create_l2_norm_operations():
+    return create_l2_norm()
```

Comment:
Can we reduce the number of files and replace methods from the `nncf.torch.graph.patterns` directly here? Since they are not used anymore, but only here.

Reply:
Done, please check